test/associations_test.rb

```diff
@@ -48,6 +48,22 @@ def test_multiple_associations
         .must_equal expected
     end
 
+    def test_nested_associations
+      peggy = Person.new
+      roger = Person.new(id: 2, name: 'Roger', occupation: 'Chief')
+      daniel = Person.new(id: 3, name: 'Daniel', occupation: 'Agent')
+      peggy.colleagues = [daniel]
+      daniel.superior = roger
+      expected = {
+        colleagues: Collection.new([daniel], root: false, includes: :superior)
+          .as_json
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.